### ISSUE TITLE
WLAN AP is not working

### ISSUE THREAD

**Issue by Bertram-561** [omersiar/esp-rfid#548]
I bought the esp-rfid Blue Board and powered it on and no Access Point to connect showed up. I flashed the firmware once again , still no AP. Is the board broken  or am I missing something ?
Regards 
Bertram

**Comment by matjack1** [comment]
hey @Bertram-561 have you tried flashing a dev version? You can try the last one from here: https://github.com/esprfid/esp-rfid/actions/runs/3404792413

@nardev do you confirm that the latest dev build works on your blue board?

**Comment by majuss** [comment]
I can confirm the issue. Used every bin available also the dev snapshot from the pipeline. Flashed with:
`esptool.py --port /dev/ttyUSB0 write_flash 0x1000 generic.bin`
No hotspot is opening, also tried the debug build, I can't see any serial output on `screen /dev/ttyUSB0 9600`.

**Comment by nardev** [comment]
The builds are same as a year ago. I usually don't flash latest FW, just slightly customized.

@Bertram-561 you can always contact me via email. Also, please confirm, was the blue led on the board on after the first powering it up?

Could be some contacts on the eps-rfid blue were not soldered well. We had one bad batch in September.

**Comment by matjack1** [comment]
@majuss if you don't see anything on the serial with the debug build, seems to me like a hardware problem :(

**Comment by nardev** [comment]
Well, as i said, it could be that the EPS doesn't boot at all.. But than the programming would also be a problematic...

**Comment by majuss** [comment]
Yes it's not booting. It's also my guess. I'am using a generic WEMOS type board.

**Comment by nardev** [comment]
Any other arduino sketch works fine?

**Comment by majuss** [comment]
Yes ESPHome for example works. Blue LED is lit up constanly.

Do I need connected hardware (in my case wiegand) while booting?

Now I'am getting an endless loop of gibberish in the monitor. Its always the sam output.

**Comment by majuss** [comment]
Update on my side. I flashed with address 0x00000 like in the flash.bat and now it worked fine! Maybe include a flashing example in the README.

Is there a way to grab other wiegand events like doorbell, or pincode entries on a keypad? I have this keypad: https://de.aliexpress.com/item/1005004210605222.html

**Comment by matjack1** [comment]
Hey @majuss thank you for the feedback!

I'm just using Platformio so I never flashed like you did. Would be nice to have a script like the flash.bat for Linux/Mac, if you want to give it a go I'm surely going to review it, thank you!
